test/helpers/click-by-text.js

```diff
@@ -2,11 +2,9 @@ module.exports = function (selector, text, browser) {
   return browser.execute(function (selector, text) {
     var elements = document.querySelectorAll(selector)
 
-    console.log(selector, text, elements.length)
-
     for (var i = 0; i < elements.length; ++i) {
       var textContent = elements[i].textContent
-      if (textContent === text) return elements[i].click()
+      if (textContent && textContent.indexOf(text) > -1) return elements[i].click()
```

Comment:
The text content includes the date and price.  I suppose we should include these in the test string as well then.